config/data_sources/reshare/03_directory_entries.yml

```diff
@@ -23,6 +23,7 @@ column_mappings:
   "de_type_rv_fk": de_type_rv_fk
   "de_published_last_update": de_published_last_update
   "de_branding_url": de_branding_url
+  "__start": __start
```

Comment:
When we ingest via CSV, any headers like __start will be automatically converted to "start". So having a column that starts with not a letter will cause issues down the line.